docs/snippets/type_syntax/color.md

```diff
@@ -0,0 +1,9 @@
+The legal values for a [`<color>`](/css_types/color) depend on the [class `Color`][textual.color.Color] and include:
```

Comment:
I don't think we need the reference to the Color class here.

```suggestion
A [`<color>`](/css_types/color) should be in one of the following formats:
```

Reply:
Fixed in 1ef20eb.

Is it worth adding a formal syntax description of the values accepted, much like we have for the styles?
Something like

```
color: named-color
     | hex-rgb
     | functional-rgb
     | functional-hsl

hex-rgb: # hex hex hex
       | # hex hex hex hex
       | # hex hex hex hex hex hex
       | # hex hex hex hex hex hex hex hex
...
```